dnSpy/dnSpy/Search/FrameworkFileUtils.cs

```diff
@@ -457,7 +457,7 @@ static class FrameworkFileUtils {
 
 		public static bool IsFrameworkAssembly(string filename, string? assemblySimpleName) {
 			// Check if it's in one of the .NET runtime dirs
-			if (Path.GetDirectoryName(Path.GetDirectoryName(Path.GetDirectoryName(filename))) is string baseDir && Directory.Exists(Path.Combine(baseDir, "Microsoft.NETCore.App")))
+			if (! string2.IsNullOrWhiteSpace(filename) && Path.GetDirectoryName(Path.GetDirectoryName(Path.GetDirectoryName(filename))) is string baseDir && Directory.Exists(Path.Combine(baseDir, "Microsoft.NETCore.App")))
```

Comment:
Without the space between the negation operator (`!`) and the call to `IsNullOrWhiteSpace` to match the rest of the dnSpy codebase.
```suggestion
			if (!string2.IsNullOrWhiteSpace(filename) && Path.GetDirectoryName(Path.GetDirectoryName(Path.GetDirectoryName(filename))) is string baseDir && Directory.Exists(Path.Combine(baseDir, "Microsoft.NETCore.App")))
```